libraries/ipUtils/ipUtils.js

```diff
@@ -0,0 +1,30 @@
+export function scrubIP(ip) {
```

Comment:
since this is used by core, it makes more sense to include it in core. (a library used by core would be included in every bundle regardless).